react-custom-hooks/client/src/components/useTodos.js

```diff
@@ -19,8 +19,12 @@ export function useTodos() {
      * and set them into the `todos` state.
      */
     async function getTodos() {
-      const data = await readTodos();
-      setTodos(data);
+      try {
+        const data = await readTodos();
+        setTodos(data);
+      } catch (err) {
+        console.error(err);
```

Comment:
There's an 'error' state variable that you should set. If you just log it, the user never finds out that something went wrong.